NBXplorer/LiquidRepository.cs

```diff
@@ -206,6 +208,30 @@ protected override async Task AfterMatch(TrackedTransaction tx, IReadOnlyCollect
 			}
 		}
 
+		
+		public override async Task<KeyPathInformation> GetUnused(DerivationStrategyBase strategy, DerivationFeature derivationFeature, int n, bool reserve)
```

Comment:
This hack will probably have side effect which will be worse than just considering blinded and unblinded as different wallet.

I don't understand why you need to do this. No user will want one store in blinded and the other unblinded and share the same indexes.

Reply:
It's more for the case that a user starts off with one option and then switches to the other, only to find out that there is address reuse happening and that some calls to GetUnused would actually return indexes that have been used. 

Maybe I am overthinking this?